### DISCUSSION TITLE
Image History Deletion

### DISCUSSION THREAD

**Discussion by plasmasolutions** [mcmonkeyprojects/SwarmUI#245] [Q&A]
Is there any way to mass delete selected images from the image history? I tried to delete it from the filesystem directly and they disappear after a refresh, but I assume that their metadata is still saved in the database, even if their key (the image) is not there anymore.

Deleting it one by one is extremely tedious...

Cheers & keep up the awesome work!

**Comment by FurkanGozukara** [comment]
I would like this too. Mass select with filter etc and delete :D

**Comment by jwax33** [comment]
In the interim, if you go to Utilities in the Server tab, there is a button called Reset All Metadata.  After you mass delete image files, hit that button and it will clear out all metadata and proceed to reload it from whatever files are remaining.

**Comment by mcmonkey4eva** [comment]
Relevant feature request is at https://github.com/mcmonkeyprojects/SwarmUI/issues/10
